### DISCUSSION TITLE
Tsoa.json config

### DISCUSSION THREAD

**Discussion by hurovroman** [hurovroman/student_workflow_organiaer#1] [Announcements]
Now I am on process with node backend and there are some problems.
Do you know how to config nodemon tsoa.json ?

**Comment by mythofmidas** [comment]
Hello.
You can cofig tsoa json simply.
Look at this solution.

![image](https://github.com/user-attachments/assets/4b20d717-804e-40cf-a6fc-d8c7dfaba4a7)

Enjoy yourself~

**Reply by hurovroman** [reply]
I works well. Thanks
